src/server/src/test/java/io/cassandrareaper/acceptance/BasicSteps.java

```diff
@@ -292,22 +292,27 @@ public void an_add_cluster_request_is_made_to_reaper() throws Throwable {
     synchronized (BasicSteps.class) {
       final StringBuffer responseData = new StringBuffer();
 
-      RUNNERS.parallelStream().forEach(runner -> {
-        Map<String, String> params = Maps.newHashMap();
-        params.put("seedHost", TestContext.SEED_HOST);
-        Response response = runner.callReaper("POST", "/cluster", Optional.of(params));
-
-        Assertions.assertThat(ImmutableList.of(
-              Response.Status.CREATED.getStatusCode(),
-              Response.Status.FORBIDDEN.getStatusCode()))
-            .contains(response.getStatus());
-
-        if (Response.Status.CREATED.getStatusCode() == response.getStatus()) {
-          responseData.append(response.readEntity(String.class));
-          Map<String, Object> cluster = SimpleReaperClient.parseClusterStatusJSON(responseData.toString());
-          TestContext.TEST_CLUSTER = (String) cluster.get("name");
-        }
-      });
+      RUNNERS
+          .parallelStream()
+          .forEach(
+              runner -> {
+                Map<String, String> params = Maps.newHashMap();
+                params.put("seedHost", TestContext.SEED_HOST);
+                Response response = runner.callReaper("POST", "/cluster", Optional.of(params));
+
+                Assertions.assertThat(
+                        ImmutableList.of(
+                            Response.Status.CREATED.getStatusCode(),
+                            Response.Status.NOT_MODIFIED.getStatusCode()))
+                    .contains(response.getStatus());
+
+                if (Response.Status.CREATED.getStatusCode() == response.getStatus()) {
+                  responseData.append(response.readEntity(String.class));
+                  Map<String, Object> cluster =
+                      SimpleReaperClient.parseClusterStatusJSON(responseData.toString());
+                  TestContext.TEST_CLUSTER = (String) cluster.get("name");
+                }
+              });
```

Comment:
what changed here?

Reply:
The FORBIDDEN status code was traded for a NOT_MODIFIED one as subsequent attempts to create a cluster will update its seed node list (and if it's the same as before, it will return NOT_MODIFIED).

Reply:
`NOT_MODIFIED` isn't mean to be used in this situation. It is only for responding to conditional-GETS. 

This has been addressed in https://github.com/thelastpickle/cassandra-reaper/pull/401